cv_next/app/cv/[cvId]/components/cvData.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+import Categories from "@/types/models/categories";
+import Image from "next/image";
+import profileIcon from "@/public/images/profile.png";
+
+export default function CvData({ cv }: { cv: CvModel }) {
+  const uploader = JSON.parse(JSON.stringify(cv.user_id));
+  const userName =
+    uploader.username && uploader.username.length > 0
+      ? uploader.username
```

Comment:
This can be simplified:
`uploader.username?.length ? uploader.username : uploader.full_name`